src/plugins/orangehrmMaintenancePlugin/config/gdpr_purge_candidate_strategy.yaml

```diff
@@ -0,0 +1,6 @@
+Entities :
+  Candidate:
```

Comment:
How about the candidate attachment and history. Does cascade remove?

Reply:
Yes. Test cases are added to service test